comicScraper.py

```diff
@@ -26,6 +26,7 @@
 readType = "&readType=1" # Suffix for issue URLs that makes it show all images on one page
 script_dir = os.path.dirname(__file__)
 COUNTER = 0 # Image Numbers
+downloadFull = False
```

Comment:
we should look into getting rid of global variables, just a thought.

Reply:
track them via input/return passing

Reply:
Passing arguments into main(....) now